### PR TITLE
feat: start game BE

### PR DESCRIPTION
**Pull request by cindyhalim**
QA Results:
```
{
 "pk": "room_X5UOX2",
 "sk": "game_FuXFr22LkJC4otDeyVeK_",
 "game": {
  "currentPlayerId": "QETpVe7SiYcAdCg=",
  "drawPile": [
   {
    "type": "playing",
    "symbol": "PAUSE",
    "text": "Ice cream flavour"
   },
   {
    "type": "playing",
    "symbol": "WAVE",
    "text": "African language"
   },
   {
    "type": "playing",
    "symbol": "WAVE",
    "text": "South American dish"
   },
   {
    "type": "playing",
    "symbol": "DOTS",
    "text": "Car sharing service"
   },
   {
    "type": "playing",
    "symbol": "HASH",
    "text": "Actress"
   },
   {
    "type": "wildCard",
    "symbols": [
     "HASH",
     "PAUSE"
    ]
   },
   {
    "type": "playing",
    "symbol": "CIRCLE",
    "text": "Sports shoe brand"
   },
   {
    "type": "playing",
    "symbol": "DIAMOND",
    "text": "Sega character"
   },
   {
    "type": "playing",
    "symbol": "WAVE",
    "text": "Greek god"
   },
   {
    "type": "playing",
    "symbol": "PLUS",
    "text": "Fast food restaurant"
   },
   {
    "type": "playing",
    "symbol": "WAVE",
    "text": "Adam Sandler Film"
   },
   {
    "type": "playing",
    "symbol": "STAR",
    "text": "Projectile weapon"
   },
   {
    "type": "playing",
    "symbol": "HASH",
    "text": "Canyon "
   },
   {
    "type": "playing",
    "symbol": "CIRCLE",
    "text": "Religious holiday"
   },
   {
    "type": "playing",
    "symbol": "DIAMOND",
    "text": "Poem"
   },
   {
    "type": "playing",
    "symbol": "HASH",
    "text": "Male name"
   },
   {
    "type": "wildCard",
    "symbols": [
     "STAR",
     "DOTS"
    ]
   },
   {
    "type": "playing",
    "symbol": "DOTS",
    "text": "Hormone"
   },
   {
    "type": "playing",
    "symbol": "DIAMOND",
    "text": "Engineering Firm"
   },
   {
    "type": "playing",
    "symbol": "PAUSE",
    "text": "Jewellery"
   },
   {
    "type": "playing",
    "symbol": "CIRCLE",
    "text": "Processed food"
   },
   {
    "type": "playing",
    "symbol": "DIAMOND",
    "text": "YouTuber"
   },
   {
    "type": "playing",
    "symbol": "PAUSE",
    "text": "Cold drink"
   },
   {
    "type": "playing",
    "symbol": "DOTS",
    "text": "Wristwatch brand"
   },
   {
    "type": "wildCard",
    "symbols": [
     "DOTS",
     "PLUS"
    ]
   },
   {
    "type": "playing",
    "symbol": "STAR",
    "text": "Vintage car"
   },
   {
    "type": "playing",
    "symbol": "CIRCLE",
    "text": "Jedi"
   },
   {
    "type": "playing",
    "symbol": "STAR",
    "text": "Endangered species"
   },
   {
    "type": "playing",
    "symbol": "CIRCLE",
    "text": "Marsupial"
   },
   {
    "type": "playing",
    "symbol": "PLUS",
    "text": "Book genre"
   },
   {
    "type": "playing",
    "symbol": "STAR",
    "text": "Bird of Prey"
   },
   {
    "type": "playing",
    "symbol": "CIRCLE",
    "text": "Birthstone "
   },
   {
    "type": "playing",
    "symbol": "DIAMOND",
    "text": "Onomatopoeia"
   },
   {
    "type": "playing",
    "symbol": "WAVE",
    "text": "Evil corporation in fiction"
   },
   {
    "type": "playing",
    "symbol": "PLUS",
    "text": "Café"
   },
   {
    "type": "playing",
    "symbol": "HASH",
    "text": "River"
   },
   {
    "type": "playing",
    "symbol": "PLUS",
    "text": "Pizza chain/brand"
   },
   {
    "type": "playing",
    "symbol": "DIAMOND",
    "text": "Sci-fi novelist"
   },
   {
    "type": "playing",
    "symbol": "PLUS",
    "text": "Traffic sign"
   },
   {
    "type": "playing",
    "symbol": "DOTS",
    "text": "Dictator"
   },
   {
    "type": "playing",
    "symbol": "WAVE",
    "text": "Irrational number"
   },
   {
    "type": "playing",
    "symbol": "DOTS",
    "text": "Beatles song"
   },
   {
    "type": "playing",
    "symbol": "CIRCLE",
    "text": "European dish"
   },
   {
    "type": "playing",
    "symbol": "CIRCLE",
    "text": "Queen"
   },
   {
    "type": "wildCard",
    "symbols": [
     "DIAMOND",
     "WAVE"
    ]
   },
   {
    "type": "playing",
    "symbol": "PAUSE",
    "text": "Caffeinated drink"
   },
   {
    "type": "playing",
    "symbol": "DIAMOND",
    "text": "Romantic era musician"
   },
   {
    "type": "playing",
    "symbol": "DOTS",
    "text": "Sea Creature"
   },
   {
    "type": "playing",
    "symbol": "DIAMOND",
    "text": "Agricultural machinery"
   },
   {
    "type": "playing",
    "symbol": "STAR",
    "text": "Yogurt brand"
   },
   {
    "type": "playing",
    "symbol": "CIRCLE",
    "text": "Mountain range"
   },
   {
    "type": "playing",
    "symbol": "PLUS",
    "text": "Internet browser"
   },
   {
    "type": "wildCard",
    "symbols": [
     "PAUSE",
     "DIAMOND"
    ]
   },
   {
    "type": "playing",
    "symbol": "DIAMOND",
    "text": "Chess opening"
   },
   {
    "type": "playing",
    "symbol": "WAVE",
    "text": "Shade of red"
   },
   {
    "type": "playing",
    "symbol": "PAUSE",
    "text": "Shape"
   },
   {
    "type": "playing",
    "symbol": "STAR",
    "text": "Ancient civilization"
   },
   {
    "type": "playing",
    "symbol": "DOTS",
    "text": "Archer"
   },
   {
    "type": "playing",
    "symbol": "DOTS",
    "text": "Petroleum Product"
   },
   {
    "type": "playing",
    "symbol": "WAVE",
    "text": "Historical general"
   },
   {
    "type": "playing",
    "symbol": "HASH",
    "text": "Burger"
   },
   {
    "type": "playing",
    "symbol": "STAR",
    "text": "Fashion designer"
   },
   {
    "type": "wildCard",
    "symbols": [
     "WAVE",
     "STAR"
    ]
   },
   {
    "type": "playing",
    "symbol": "WAVE",
    "text": "Anime series"
   },
   {
    "type": "playing",
    "symbol": "PLUS",
    "text": "Allergy"
   },
   {
    "type": "playing",
    "symbol": "STAR",
    "text": "Star Trek Character"
   },
   {
    "type": "playing",
    "symbol": "STAR",
    "text": "Venomous snake"
   },
   {
    "type": "playing",
    "symbol": "CIRCLE",
    "text": "Magical item"
   },
   {
    "type": "wildCard",
    "symbols": [
     "CIRCLE",
     "HASH"
    ]
   },
   {
    "type": "playing",
    "symbol": "PLUS",
    "text": "Cartoon"
   },
   {
    "type": "playing",
    "symbol": "PAUSE",
    "text": "Cookie"
   },
   {
    "type": "playing",
    "symbol": "WAVE",
    "text": "Engine part"
   },
   {
    "type": "playing",
    "symbol": "PLUS",
    "text": "Mythological creature"
   },
   {
    "type": "playing",
    "symbol": "PLUS",
    "text": "Planet"
   },
   {
    "type": "playing",
    "symbol": "CIRCLE",
    "text": "Shoe brand"
   },
   {
    "type": "playing",
    "symbol": "DOTS",
    "text": "Fictional technology"
   },
   {
    "type": "wildCard",
    "symbols": [
     "PLUS",
     "CIRCLE"
    ]
   },
   {
    "type": "playing",
    "symbol": "HASH",
    "text": "Water sport"
   },
   {
    "type": "playing",
    "symbol": "PLUS",
    "text": "School supply item"
   },
   {
    "type": "playing",
    "symbol": "STAR",
    "text": "Country song"
   },
   {
    "type": "playing",
    "symbol": "WAVE",
    "text": "Eastern Philosopher"
   },
   {
    "type": "playing",
    "symbol": "STAR",
    "text": "Middle Eastern country"
   },
   {
    "type": "playing",
    "symbol": "DIAMOND",
    "text": "Tsar"
   },
   {
    "type": "playing",
    "symbol": "CIRCLE",
    "text": "Negative emotion"
   },
   {
    "type": "playing",
    "symbol": "PAUSE",
    "text": "Carnival ride"
   },
   {
    "type": "playing",
    "symbol": "PAUSE",
    "text": "Sea"
   },
   {
    "type": "playing",
    "symbol": "PAUSE",
    "text": "Phone brand"
   },
   {
    "type": "playing",
    "symbol": "WAVE",
    "text": "Fictional language"
   },
   {
    "type": "playing",
    "symbol": "PAUSE",
    "text": "War"
   },
   {
    "type": "playing",
    "symbol": "DIAMOND",
    "text": "Programming language"
   },
   {
    "type": "playing",
    "symbol": "STAR",
    "text": "King"
   },
   {
    "type": "playing",
    "symbol": "DIAMOND",
    "text": "Heist film"
   },
   {
    "type": "playing",
    "symbol": "DOTS",
    "text": "Doomsday scenario"
   },
   {
    "type": "playing",
    "symbol": "PAUSE",
    "text": "Winter clothing"
   },
   {
    "type": "playing",
    "symbol": "HASH",
    "text": "Natural disaster"
   },
   {
    "type": "playing",
    "symbol": "PAUSE",
    "text": "Sandwich"
   },
   {
    "type": "playing",
    "symbol": "DOTS",
    "text": "21st century innovation"
   },
   {
    "type": "playing",
    "symbol": "DOTS",
    "text": "Muscle"
   },
   {
    "type": "playing",
    "symbol": "PLUS",
    "text": "Fabric"
   },
   {
    "type": "playing",
    "symbol": "HASH",
    "text": "Highschool"
   }
  ],
  "id": "FuXFr22LkJC4otDeyVeK_",
  "players": [
   {
    "name": "Shabu Shabu",
    "id": "QETpVe7SiYcAdCg=",
    "isAdmin": true,
    "winningPile": [],
    "playPile": []
   },
   {
    "name": "Tofu",
    "id": "QETq2eq5iYcCH2w=",
    "isAdmin": false,
    "winningPile": [],
    "playPile": []
   },
   {
    "name": "Sukiyaki",
    "id": "QETqCfN4iYcAcYw=",
    "isAdmin": false,
    "winningPile": [],
    "playPile": []
   }
  ],
  "wildCardPile": [],
  "status": "started"
 }
}
```

### CODE REVIEW

File: backend/src/functions/onGameStarted.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  const message = players.map((player) =>
+    ws.sendMessage<IGameUpdatedEvent>({
+      connectionId: player.id,
+      body: {
+        action: BackendWebsocketActions.GameUpdated,
+        currentGame: {
+          id: newGame.id,
+          players: newGame.players,
+          totalDrawCardsRemaining: newGame.drawPile.length,
+          wildCard: newGame.wildCardPile[newGame.wildCardPile.length - 1],
```
Comment (emilyhfdong) [inline backend/src/functions/onGameStarted.ts]: should we start the game with no wild card?

File: backend/src/helpers/__tests__/cards.tests.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
+import { CardsDataJSON } from "@services"
```
Comment (emilyhfdong) [inline backend/src/helpers/__tests__/cards.tests.ts:1]: :its_so_beautiful:

File: backend/src/helpers/cards.ts [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+      symbolSet.add(JSON.stringify(chosenSymbols))
+      symbolSet.add(JSON.stringify(flippedChosenSymbols))
+    } else {
+      shuffledSymbolB.push(symbolB)
+      shuffledSymbolA.push(symbolA)
+      shuffledSymbolB = shuffle(shuffledSymbolB)
+    }
+  }
+
+  const drawPile = shuffle([...unshuffledDrawPile, ...wildCardPile])
```
Comment (cindyhalim) [inline backend/src/helpers/cards.ts:83]: updated the logic here– on game start, draw pile returns 92 playing cards and 8 wild cards shuffled and wild card pile will be empty. 

`card_drawn` event can then handle moving wild cards from draw pile to wild card pile

**Review by emilyhfdong — APPROVED** [review]
LGTM!!!!!

**Pull request merged**